docs/deployment/semgrep-app-ghe.md

```diff
@@ -0,0 +1,102 @@
+---
+slug: semgrep-app-ghe
+append_help_link: true
+title: Install the Semgrep App for GitHub Enterprise
+hide_title: false
+description: Learn how to install the Semgrep App for use |
+    with GitHub Enterprise (GHE) deployments.
+tags:
+  - GitHub Enterprise
+  - Semgrep App
+---
+
+The Semgrep App for GitHub Enterprise (GHE) creates a connection between Semgrep
+and orgs in your GHE deployment. It allows you to get Semgrep results as PR
+comments, add Semgrep to your projects with one click, and manage rules and
+results across multiple projects from one centralized place.
+
+This article walks you through installing the Semgrep App for use with GHE
+deployments. There are two primary installation steps:
+
+1. Install the Semgrep App for the first time using the GHE organization (org)
+   that "owns" the app.
+2. Install the app for additional GHE orgs.
+
+## Install the Semgrep App for the first time using the organization that "owns" the app
+
+When you install the Semgrep App for GHE for the first time in your GHE
+instance, you must choose the organization within your deployment to act as the
+owner of the app. As the owner, this GHE org controls the settings and
+permissions granted to the app.
+
+1. Log in to [Semgrep Cloud Platform](https://semgrep.dev/login/).
+1. Go to **Settings** > **Source Code Managers**, and click **Add GitHub
+   Enterprise**.
+   ![Semgrep Cloud Platform's Source code managers page](/img/ghe-1.png#md-width)
+1. In the popup window, provide:
+   - The **Name of your GitHub Organization**
+   - The **URL** to access your deployment
+   - The **Access token** Semgrep can use to post pull request comments to your
```

Comment:
In [the doc](https://docs.google.com/document/d/1TwFm5jSs_a2LwIW3CSsnzNa_vokkPQOc3xGRgBr-2i4/edit), it states that the access token is optional -- why are we filling this out?

Reply:
Rather, it's not optional, but it's not used in this method.

Reply:
I made a mistake when editing the doc